src/repositories/user.repository.js

```
@@ -24,7 +24,7 @@ module.exports = class UserRepository {
                 }
             }
             else{
-                throw new Error("password doesn't match");
+                console.log("password doesn't match");
```

I think you still want to throw an error here? But want to make sure it gets caught by your api and returned to the client.